locales/en-US/common.ftl

```diff
@@ -62,5 +62,3 @@ common-moderationReason-detailedExplanation-placeholder =
 
 common-userBanned =
```

Comment:
Looks like we probably need to move this one over to the server common translation file too?